Grid/qcd/action/gauge/Gauge.h

```diff
@@ -42,26 +45,33 @@ typedef WilsonGaugeAction<PeriodicGimplD>          WilsonGaugeActionD;
 typedef WilsonGaugeAction<SpPeriodicGimplR>        SpWilsonGaugeActionR;
 typedef WilsonGaugeAction<SpPeriodicGimplF>        SpWilsonGaugeActionF;
 typedef WilsonGaugeAction<SpPeriodicGimplD>        SpWilsonGaugeActionD;
+
+//typedef LLRWilsonGaugeAction<PeriodicGimplR>          LLRWilsonGaugeActionR;
```

Comment:
Don't introduce new commented-out code. Either revert this change, or uncomment it, and also add `D` and `F`-suffixed versions.